### DISCUSSION TITLE
When will v3 be launched?

### DISCUSSION THREAD

**Discussion by bfig** [requarks/wiki#6433] [Help / Questions] [answered]
Hello there,

I have been loving the product and would love to manage my expectations regarding your schedule and objectives for this stable v3. It seems like there are already a significant amount of features and I was wondering if there is a release date planned.

Best regards,
Bruno.

**Comment by NGPixel** [comment, marked as answer]
https://docs.requarks.io/releases/eta

https://blog.js.wiki/news?tag=3.x
